src/var/isWindow.js

```diff
@@ -0,0 +1,8 @@
+define( function() {
+	"use strict";
+
+	return function isWindow( obj ) {
+		return obj !== null && typeof obj !== "undefined" && obj === obj.window;
```

Comment:
@mgol Got it, I've retained `obj!=null` instead of `obj!==null` and made a commit. The tests all seem to pass with AMD mode enabled.

<img width="813" alt="screen shot 2017-06-25 at 9 55 58 am" src="https://user-images.githubusercontent.com/650317/27513656-8573755e-598c-11e7-9e55-bfa0dae59d5d.png">